components/ui/AppMegaDropdownMenu.vue

```diff
@@ -88,59 +240,68 @@ export default class TheMegaDropdownMenu extends Vue {
   color: $cool-gray-80;
   position: relative;
 
-  &__button {
-    &_primary,
-    &_secondary {
+  &__search-field-wrapper {
+    display: flex;
+    width: 18rem;
+    height: 2.5rem;
+    justify-content: space-between;
+    align-items: center;
+    padding: $spacing-03;
+
+    color: $cool-gray-80;
+    fill: $cool-gray-80;
```

Comment:
We have new color variables for this:

```suggestion
    color: $text-color-light;
    fill: $text-color-light;
```

Reply:
We should rebase the textbook demo branch in order to use those variables.

Reply:
#1839